phoenix-core/src/main/java/org/apache/phoenix/schema/types/PDouble.java

```diff
@@ -17,15 +17,16 @@
  */
 package org.apache.phoenix.schema.types;
 
-import com.google.common.base.Preconditions;
-import com.google.common.primitives.Doubles;
```

Comment:
I've already imported the Import Order Config Files, and the rule is:

java
org
com

That's why my eclipse changes the order.